packages/inula-adapter/examples/$parent/list.jsx

```
@@ -0,0 +1,27 @@
+
+  expose(setFilterType);
+
+  return (
+    <table>
+      <tr>
+        <td>ID</td>
+        <td>Product name</td>
+        <td>Product type</td>
+      </tr>
+      {filteredIdems.map(item => (
+        <TableItem productId="item.id" productName="item.name" productType="item.type"></TableItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)